app/gui/src/dashboard/components/Devtools/EnsoDevtoolsImpl.tsx

```diff
@@ -350,27 +386,85 @@ export function EnsoDevtools() {
             />
           </div>
 
-          <div className="flex flex-col gap-0.5">
-            {LocalStorage.getAllKeys().map((key) => (
-              <div key={key} className="flex w-full items-center justify-between gap-1">
-                <Text variant="body">
-                  {key
-                    .replace(/[A-Z]/g, (m) => ' ' + m.toLowerCase())
-                    .replace(/^./, (m) => m.toUpperCase())}
-                </Text>
-
-                <Button
-                  variant="icon"
-                  size="small"
-                  isDisabled={localStorageState[key] == null}
-                  aria-label={getText('delete')}
-                  icon={CrossIcon}
-                  onPress={() => {
-                    localStorage.delete(key)
-                  }}
-                />
-              </div>
-            ))}
+          <div className="flex flex-col gap-1.5">
+            {LocalStorage.getAllKeys().map((key) => {
+              const metadata = LocalStorage.getKeyMetadata(key)
+              const title = key
+                .replace(/[A-Z]/g, (m) => ' ' + m.toLowerCase())
+                .replace(/^./, (m) => m.toUpperCase())
+
+              return (
+                <div key={key} className="flex w-full items-center justify-between gap-1">
+                  <div className="flex items-center gap-1">
+                    <Text variant="body">{title}</Text>
+
+                    <VisualTooltip tooltip="User specific storage item">
+                      {metadata.isUserSpecific === true && (
+                        <Icon icon="default_user" size="small" />
+                      )}
+                    </VisualTooltip>
+                  </div>
+
+                  <Button.Group
+                    align="end"
+                    buttonVariants={{ size: 'small', variant: 'icon', extraClickZone: 'small' }}
+                  >
+                    <Dialog.Trigger>
+                      <Button aria-label="Edit" icon={EditIcon} />
```

Comment:
```suggestion
                      <Button aria-label="Edit" icon="edit" />
```
Let's keep moving towards shared icons everywhere so we can one day remove the individual svgs.

Reply:
@AdRiley is it possible to update the edit icon or add a new one from here: https://www.figma.com/design/4vAG8Oino0kNO5fhKDiCG6/central%2Bicon%2Bsystem%2Bv1.17?node-id=7632-66247&t=JZyWXr77w3Y8jBi0-4) ?

Reply:
Yep https://github.com/enso-org/enso/pull/12494

Reply:
Merged

Reply:
@MrFlashAccount does this still need to be changed to a shared icon?

Reply:
Yes, I'll replace it with all the addresses to QA/CR